docs/user-guide/admin-cli.md

```diff
@@ -408,7 +408,7 @@ optional arguments:
 Grants datastore access to a vmgroup.
 Valid value for "datastore" includes the name of valid datastores in the ESX host , special value "_VM_DS" or "_ALL_DS".
 When DS is set to _VM_DS, access to vm_datastore where vm lives is allowed for vms in vmgroup.
-When DS is set to _ALL_DS, access to all DS is allowed for vms in vmgroup.
+When DS is set to _ALL_DS, access to any other datastore which has not been granted access to explicitly is allowed for vms in vmgroup.
```

Comment:
Why "any other datastore"? Do you mean the datastore where vm lives is NOT allowed? That seems counter-intuitive.